build/vars.build.mk

```diff
@@ -14,15 +13,14 @@ LOCAL_AM_CFLAGS = \
 	-DDOCDIR=\""$(docdir)"\" \
 	-DGEANYPLUGINS_DATADIR=\""$(datadir)"\" \
 	-DPKGDATADIR=\""$(pkgdatadir)"\" \
-	-DLIBDIR=\""$(libdir)"\" \
-	-DPKGLIBDIR=\""$(pkglibdir)"\"
```

Comment:
`pkglibdir` is `$(libdir)/$(PACKAGE)`.

And well, it's a common thing to have but yeah as it's not used I guess we can remove it.